openslides/agenda/views.py

```diff
@@ -317,6 +317,7 @@ def manage_speaker(self, request, pk=None):
         """
         # Retrieve list of speakers.
         list_of_speakers = self.get_object()
+        responseData = None
```

Comment:
Please undo changes here and do a seperate request endpoint to do the new operation. Also the server uses snake case (lower cases with underscores) as the naming conventions for variables and attributes.